docs/contributing.rst

```diff
@@ -267,6 +267,17 @@ Please:
 .. _PEP 8 - Style Guide for Python Code:
   https://www.python.org/dev/peps/pep-0008
 
+Submitting a pull request
+=========================
+Steps:
+
+1. Write your code!
+2. Make sure your environment is set up properly and that you're in your virtualenv (this is a **very important** step).
```

Comment:
1. please wrap lines
2. tell the reader to use `./bootstrap/dev/venv.sh` and activate the virtualenv?